Makefile

```diff
@@ -9,8 +9,8 @@ docker-running:
 	systemctl start docker
 
 TAG := $(shell date --iso-8601)
-TEST_SECRETS := /var/lib/cockpit-tests/secrets
-TEST_CACHE := /var/cache/cockpit-tests
+TEST_SECRETS := /var/lib/cockpit-tasks/secrets
+TEST_CACHE := /var/cache/cockpit-tasks
```

Comment:
Let's rename the variables to `TASKS_*`, too. When deploying this container, we'll need to adjust this path on our bos cluster. We can do this now and add a tests → tasks symlink. (Creating a TODO item for that)